plugins/modules/ovirt_vm.py

```diff
@@ -1784,7 +1784,7 @@ def check_custom_compatibility_version():
             equal(self.param('serial_policy'), str(getattr(entity.serial_number, 'policy', None))) and
             equal(self.param('serial_policy_value'), getattr(entity.serial_number, 'value', None)) and
             equal(self.param('numa_tune_mode'), str(entity.numa_tune_mode)) and
-            equal(self.param('virtio_scsi_enabled'), entity.virtio_scsi.enabled) and
+            equal(self.param('virtio_scsi_enabled'), getattr(entity.virtio_scsi, 'enabled', None)) and
             equal(self.param('multi_queues_enabled'), entity.multi_queues_enabled) and
```

Comment:
don't we need to do something similar here as well?

Reply:
I don't think so, as multi_queues_enabled for example always exists in the entity.
For virtio_scsi we need this, because the object is not always created, and thus enabled is checked of None.

See https://github.com/oVirt/ovirt-ansible-collection/blob/dfb33536f7dcc73870aa6db9657775522669b67b/plugins/modules/ovirt_vm.py#L1585

Reply:
But we might need it in some other places:

https://github.com/oVirt/ovirt-ansible-collection/blob/dfb33536f7dcc73870aa6db9657775522669b67b/plugins/modules/ovirt_vm.py#L1765 Is one for example

But also: https://github.com/oVirt/ovirt-ansible-collection/blob/dfb33536f7dcc73870aa6db9657775522669b67b/plugins/modules/ovirt_vm.py#L1771

So yes, needs some more improvements

Reply:
Fixed in new push